Dockerfile

```diff
@@ -93,18 +107,18 @@ COPY modules/conftest.py modules/conftest.py
 COPY modules/setup.cfg modules/setup.cfg
 
 # Migrate files from spot
-COPY --from=spot /usr/local/lib/*spot* /opt/venv/lib
-COPY --from=spot /usr/local/lib/*bdd* /opt/venv/lib
-COPY --from=spot /usr/local/lib/python3.10/site-packages/spot /opt/venv/lib/python3.10/site-packages/spot
-COPY --from=spot /usr/local/lib/python3.10/site-packages/*buddy* /opt/venv/lib/python3.10/site-packages/
+COPY --from=spot /usr/local/lib/*spot* /usr/local/lib/
+COPY --from=spot /usr/local/lib/*bdd* /usr/local/lib/
+COPY --from=spot /usr/local/lib/python3.8/site-packages/spot /usr/local/lib/python3.8/site-packages/spot
+COPY --from=spot /usr/local/lib/python3.8/site-packages/*buddy* /usr/local/lib/python3.8/site-packages/
```

Comment:
Please undo these too. This is likely a result of accepting incoming changes only during conflict resolution when rebasing your branch from main.